public/app/plugins/datasource/prometheus/datasource.test.ts

```diff
@@ -1861,9 +1863,9 @@ describe('PrometheusDatasource for POST', () => {
         timeSrvStub as any
       );
       mockDs._addTracingHeaders(httpOptions as any, options as any);
-      expect(httpOptions.headers['X-Dashboard-Id']).toBe(undefined);
-      expect(httpOptions.headers['X-Panel-Id']).toBe(undefined);
-      expect(httpOptions.headers['X-Dashboard-UID']).toBe(undefined);
+      expect(httpOptions.headers['X-Dashboard-Id']).toBe(1);
+      expect(httpOptions.headers['X-Panel-Id']).toBe(2);
+      expect(httpOptions.headers['X-Dashboard-UID']).toBe('WFlOM-jM1');
```

Comment:
if you would add the following to this `describe` section all tests will be passed.  Also, you have to remove the `access: 'proxy:'` from above. See my suggestions.
```ts
const instanceSettings = {
      url: 'proxied',
      directUrl: 'direct',
      user: 'test',
      password: 'mupp',
      access: 'proxy',
      jsonData: { httpMethod: 'POST' },
    } as unknown as DataSourceInstanceSettings<PromOptions>;

    let ds: PrometheusDatasource;
    beforeEach(() => {
      ds = new PrometheusDatasource(instanceSettings, templateSrvStub as any, timeSrvStub as any);
    });
```